Userland/Utilities/route.cpp

```diff
@@ -157,8 +157,18 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         if (!value_host_address.is_empty())
             destination = AK::IPv4Address::from_string(value_host_address);
 
-        if (!value_network_address.is_empty())
-            destination = AK::IPv4Address::from_string(value_network_address);
+        StringView address;
+        StringView cidr;
+        if (!value_network_address.is_empty()) {
+            // Check if a CIDR notation was provided and parse accordingly
+            if (auto position = value_network_address.find('/'); position.has_value()) {
+                address = value_network_address.substring_view(0, position.value());
+                cidr = value_network_address.substring_view(position.value() + 1);
```

Comment:
Does this still work for input like `1.2.3.4/`, where `/` is the last character?

Reply:
Yeah, it does. It won't pass the `cidr_int.has_value()` conditional check below, so an empty string will be passed to generate the genmask `IPv4Address`. That will also fail and then the program will exit on failure.